docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3"
+
+services:
+  backend:
+    restart: unless-stopped
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    environment:
+      NODE_ENV: production
+      PORT: 3030
+      POSTGRESQL_HOST: postgres
+      POSTGRESQL_USER: postgres
+      POSTGRESQL_PASSWORD: password
+      POSTGRESQL_DATABASE: mydb
+      JWT_SECRET: secret
+    ports:
+      - 3030:3030
+    depends_on:
+      - database
+      - postgres
+
+  database:
+    restart: on-failure
+    build:
+      context: ./database
+      dockerfile: Dockerfile
+    environment:
+      POSTGRESQL_PORT: 5432
+      POSTGRESQL_HOST: postgres
+      POSTGRESQL_USER: postgres
+      POSTGRESQL_PASSWORD: password
+      POSTGRESQL_DATABASE: mydb
+    depends_on:
+      - postgres
+
+  migration:
+    restart: unless-stopped
+    build:
+      context: ./migration
+      dockerfile: Dockerfile
+    environment:
+      AUTH_DOMAIN: ${AUTH_DOMAIN}
+      PROJECT_ID: ${PROJECT_ID}
+      STORAGE_BUCKET: ${STORAGE_BUCKET}
+      NODE_ENV: production
+      POSTGRESQL_HOST: postgres
+      POSTGRESQL_USER: postgres
+      POSTGRESQL_PASSWORD: password
+      POSTGRESQL_DATABASE: mydb
+    ports:
+      - 8080:8080
+    depends_on:
+      - database
+      - postgres
+
+  postgres:
+    image: postgres
```

Comment:
Can you specify the major version of the database that you'll be using? 14 and 15 are incompatible, so if someone had a previous version of 14 running and they'll try run 15 now, it'll fail as the versions are incompatible.

Reply:
Other than that though, looks fine.

Reply:
done :)